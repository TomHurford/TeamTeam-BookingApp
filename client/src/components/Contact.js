import React from 'react';
import '../styles/Contact.css';
//A component for Contact page to be added

function Contact(){
    return(
        <div className='page-container'>
            <div className='underlay'></div>
            <h1 className='title'>Who are we</h1>
            <p className='data'>we are an event booking application with the aim to help consumers have access to events avaiable </p>
            <p className='data'>contact email:ticketopiacontact@gmail.com</p>
        </div>
    )
}
export default Contact
