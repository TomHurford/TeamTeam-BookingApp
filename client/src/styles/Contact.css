.page-container > .title {
    margin-top: 60px;
    text-align:center;
}

.data{
    text-align:center;
    align-self: right;
    width: 70%;
    margin: auto;
}
