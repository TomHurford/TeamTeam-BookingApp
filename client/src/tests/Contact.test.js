import React from 'react';
import renderer from 'react-test-renderer';
import Contact from '../components/Contact';

test('Contact snapshot test', () => {
  const component = renderer.create(<Contact />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});