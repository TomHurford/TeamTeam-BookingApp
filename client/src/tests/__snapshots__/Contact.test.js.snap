// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact snapshot test 1`] = `
<div
  className="page-container"
>
  <div
    className="underlay"
  />
  <h1
    className="title"
  >
    Who are we
  </h1>
  <p
    className="data"
  >
    Ticketopia is...
  </p>
  <div
    style={
      Object {
        "backgroundColor": "#abc4ff",
        "borderColor": "gray",
        "borderRadius": 20,
        "borderWidth": 2,
      }
    }
  >
    <h3>
      <center>
        Contact Us
      </center>
    </h3>
    <form
      onSubmit={[Function]}
      style={
        Object {
          "marginLeft": 38,
          "width": "95%",
        }
      }
    >
      <div
        className="form-group"
      >
        <label
          className="form-label "
          htmlFor="customerName"
        >
          Your Name
        </label>
        <input
          className="form-control"
          name="customerName"
          onBlur={[Function]}
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className="form-label "
          htmlFor="customerEmail"
        >
          Your Email
        </label>
        <input
          className="form-control"
          name="customerEmail"
          onBlur={[Function]}
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className="form-label "
          htmlFor="messageSubject"
        >
          Subject of your query
        </label>
        <input
          className="form-control"
          name="messageSubject"
          onBlur={[Function]}
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className="form-label "
          htmlFor="message"
        >
          Your query
        </label>
        <textarea
          className="form-control"
          cols="173"
          id="message"
          name="message"
          onBlur={[Function]}
          onChange={[Function]}
          rows="8"
          style={
            Object {
              "marginBottom": "8px",
            }
          }
          type="text"
          value=""
        />
      </div>
      <button
        className="btn btn-primary"
        style={
          Object {
            "marginBottom": "8px",
          }
        }
      >
        Submit query
      </button>
    </form>
  </div>
</div>
`;
